apps/universal/src/app/kitchen-sink/kitchen-sink.html

```diff
@@ -301,6 +301,7 @@ <h1>Packets</h1>
   voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 </dt-expandable-text>
 
+<!--
```

Comment:
Please do not comment the event-chart here. We need to test it. If it does not work, please remove it and add a follow up issue.

Reply:
Just forgot to comment it in :)